playbooks/adhoc/README.md

```diff
@@ -0,0 +1,4 @@
+# _Ad hoc_ playbooks
+
+This directory holds playbooks and tasks that really don't have a better home.
+Existing playbooks living here are generally unsupported and not maintained.
```

Comment:
s/unsupported and not maintained/community supported and not officially maintained/